src/microbe_stage/MicrobeStage.cs

```diff
@@ -821,7 +821,10 @@ private void OnPlayerReproductionStatusChanged(Microbe player, bool ready)
         if (ready && (player.Colony == null || player.IsMulticellular))
         {
             if (!player.IsMulticellular)
+            {
+                HUD.SetupPressEditorTutorial(TutorialState);
```

Comment:
This should be done in the tutorial setup for the stage, not just before emitting the event, because that is not flexible, if the trigger condition is changed this code also needs to move so that's why this is not futureproof. This should be done similarly to how the undo/redo buttons are sent to the tutorial, which is when the editor GUI is initialized: https://github.com/andeason/Thrive/blob/070fa612f641efdd582130e639349dff8708c03a/src/microbe_stage/editor/MicrobeEditor.cs#L167

Reply:
I put in the SetupStage after setting the eventreciever as that seemed like the best place for now.  I can attempt to do it in the HUD init, but from a quick glance seems that I would need to reorder the various operations to make it work for that.  Could be wrong, though.